code-studio/src/js/external/external.js

```diff
@@ -1,5 +1,3 @@
-//= require_self
-
```

Comment:
http://stackoverflow.com/questions/6407067/what-does-require-self-mean Based on this answer, I think it's for CSS - so probably not at all useful here?